### ISSUE TITLE
Notification on Repo Update

### ISSUE THREAD

**Issue by Ineluctable** [sibbell/sibbell-issues#38]
Any chance you could have an option to notify when a repo is updated? I have a couple of Repos that I follow that don't regularly make releases, but I'd like to know when they're updated - ie, new commits have been made.

Obviously that'd be spammy on some repos, but on low-traffic repos it would be a great opt-in feature - especially if you condensed the updates into a daily email.

Thanks!

**Comment by davegaeddert** [comment]
I could certainly look into it. So what all would you ideally like to see? And how does that compare to the GitHub ability to "watch" a repo? Daily email updates would be a differentiator for sure, but other than that I'd be interested in figuring out how this would be different than watching a repo. Thoughts @Ineluctable?

They say a little bit here about what kinds of notifications you get when you watch a repo: https://help.github.com/articles/about-notifications/

**Comment by Ineluctable** [comment]
Oh cool - I didn't realize you'd get notifications for each commit by watching a repo - I figured that was more for issues and things like that. I'll give that a try!

Pretty much what I'd like is to get notifications of each commit for certain repos - so I think you're right, the only difference would be the daily emails.

Thanks for the link - I appreciate it!

**Comment by davegaeddert** [comment]
Actually, I don't think you'll get a notification for every commit -- just if there's a comment on a commit (plus the stuff for issues, PRs, releases). So you'd basically like a digest of everything that was pushed in a day? Just as a way to keep tabs on it?

**Comment by davegaeddert** [comment]
@Ineluctable also just found this: https://help.github.com/articles/receiving-email-notifications-for-pushes-to-a-repository/

I've never tried it but maybe give it a try? I'd be really curious to see if you find something that does what you want or not. Now that you mention it, I can also think of situations where I'd like to see a "digest of commits made yesterday"...

**Comment by Ineluctable** [comment]
Exactly. Some of the repositories I am interested in don't make regular releases - but I'd love to know when they get updated and have a small summary of what was changed.

That's a great find Dave, thank you - I'll give it a try!

**Comment by Ineluctable** [comment]
It looks like that will only work on repositories you have owner access to - you have to be able to access a Repo's settings to be able to follow those steps. :( It was worth a shot though!

**Comment by davegaeddert** [comment]
Ah. Sorry I've got internal projects on the brain today ;)

Sounds like there may be something here worth doing! I'll give some thought as to whether or not this fits in with the direction Sibbell is headed in -- I probably won't get to it for a little while though. If you want to write up any more specifics about how you think you'd like to see this work (commits from all branches? just master? summary of issues and PRs too?), feel free! Thanks @Ineluctable!

**Comment by Ineluctable** [comment]
For me, I'd just be happy with getting details from the main branch - just the commit summaries and a link to view.


After doing some searching, I found a way to get commit updates through a site called Zapier, you can setup a task that will send out an email for each commit (no options for a digest that I saw). It allows you to specify the repository name and branch and you can customize each message to include whatever fields from each commit you want. It's a bit cumbersome to setup (you have to create a new task for each repo) - but it'll serve me well in the interim while you decide if you want to add it to Sibbell!


Here's a view of some of the options in the setup:
![2015-11-24_02-25-21](https://cloud.githubusercontent.com/assets/8416589/11361387/ddcce7a6-9252-11e5-8d62-d2455bed575e.png)

And a screenshot of the email they sent: 
![2015-11-24_02-12-45](https://cloud.githubusercontent.com/assets/8416589/11361256/da9f1b04-9251-11e5-9021-016306640fc1.png)
